### PR TITLE
Account file nickname mandatory

### PR DESCRIPTION
**Pull request by Thykof**

### CODE REVIEW

**Review by B-Naoufal — APPROVED** [review]

File: wallet/file-format.md [hunk 1]
```diff
@@ -78,7 +79,7 @@ The following table summarize the format:
 | Field | Presence | Format | Comment | Example |
 | ----- | -------- | ------ | ------- | ------- |
 | Version | Mandatory | Integer | Entire part of this specification version | 0 |
-| Nickname | Optional | String || "Savings" |
+| Nickname | Mandatory | String || "Savings" |
```
Comment (gregLibert) [inline wallet/file-format.md:82]: @massalabs/core-team can you check that you can also follwo this constraint ?

Comment (AurelienFT) [reply wallet/file-format.md:82]: I'm currently working on using this standard on the node for now we still have our custom standard

Comment (gregLibert) [reply wallet/file-format.md:82]: > I'm currently working on using this standard on the node for now we still have our custom standard

Yes, I know. But do you think that the fact that the nickname is mandatory is blocking on your side ?

Comment (AurelienFT) [reply wallet/file-format.md:82]: No I'll just add a default one for now

**Pull request merged**